examples/lighting-app/efr32/BUILD.gn

```diff
@@ -82,7 +82,8 @@ if (chip_enable_wifi) {
 }
 
 # BRD4166A --> ThunderBoard Sense 2 (No LCD)
-if (efr32_board == "BRD4166A" || efr32_board == "BRD4180A") {
+if (efr32_board == "BRD4166A" || efr32_board == "BRD4180A" ||
+    efr32_board == "BRD4186A" || efr32_board == "BRD4187A") {
```

Comment:
Don't disable the LCD with MG24 if not needed. If the WiFi support is too big, please include a condition so that the OpenThread example are not affected.

Reply:
Yes, suggested changes have been done and added